cfgmgr/buffermgrdyn.h

```diff
@@ -79,12 +79,12 @@ typedef struct {
 } buffer_pg_t;
 
 typedef enum {
+    // Port is admin down. All PGs programmed to APPL_DB should be removed from the port
```

Comment:
Why do you need to change order of the enum? does the enum number has meaning?

Reply:
The first enum value (0) is the default value after initialization.
Originally, the default state was `PORT_INITIALIZING`.
As sometimes the `admin_status` doesn't exist in `CONFIG_DB` if the port is admin down, we need to change the default state to `PORT_ADMIN` and change a port's state to another state after it is admin state is learned.